### PR TITLE
Correct package name

### PR DESCRIPTION
**Pull request by res0nance**
- Correct the package name from #4038 

### Submitter checklist

- [ ] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
oops, did not notice it. We cannot just rename it unfortunately. It would impact binary compatibility
But we can introduce a deprecated class in `hudson.os` which just inherits the "new class"

**Comment by res0nance** [comment]
Makes sense, I'm fixing the references to it in the tests then I can add the deprecated stub after.

**Review by daniel-beck — CHANGES_REQUESTED** [review]
I would move the file to the specified (`hudson.…`) package and move on. No need to include more code for this simple mistake.

Please do not include unnecessary whitespace changes.

**Comment by oleg-nenashev** [comment]
@res0nance We do not get GitHub notifications on Force pushes. Please avoid using them, we can always squash commits during merge

File: core/src/test/java/hudson/FilePathTest.java [hunk 1]
```diff
@@ -1,18 +1,18 @@
 /*
  * The MIT License
- *
```
Comment (oleg-nenashev) [inline core/src/test/java/hudson/FilePathTest.java]: All the formatting changes are not related. Please submit refactoring separately

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
It still causes binary compatibility issues. I am ready to accept a "quick fix" sugestion by @daniel-beck where we just move the code to the right directory so that it stays in `hudson.os`. Not ideal, but we can live with that to save time

**Comment by res0nance** [comment]
I just moved the file as per @daniel-beck suggestion

**Review by oleg-nenashev — APPROVED** [review]

**Review by jeffret-b — APPROVED** [review]
It will be good to get this corrected quickly so it makes more sense.

**Review by oleg-nenashev — APPROVED** [review]

**Pull request merged**